src/cloud/project/magento-app-php-application.md

```diff
@@ -41,96 +41,19 @@ php -m
 
 For details about a specific PHP extension, see the [PHP Extension List](https://www.php.net/manual/en/extensions.alphabetical.php).
 
-{{site.data.var.ece}} supports the following extensions:
+The following table show the supported PHP extensions when deploying {{site.data.var.ee}} on the Cloud platform.
 
--  Default extensions:
-   -  `bcmath`
-   -  `bz2`
-   -  `calendar`
-   -  `exif`
-   -  `gd`
-   -  `gettext`
-   -  `intl`
-   -  `mysqli`
-   -  `pcntl`
-   -  `pdo_mysql`
-   -  `soap`
-   -  `sockets`
-   -  `sysvmsg`
-   -  `sysvsem`
-   -  `sysvshm`
-   -  `opcache`
-   -  `zip`
+- **Default extensions** are the default extensions available with the PHP application.
+- **Required extensions** are installed automatically and cannot be removed
+- **Optional extensions**–You can install and remove these extensions as needed
```

Comment:
```suggestion
- **Optional extensions**–You can install and remove these extensions as needed.
```

Reply:
Removed these updates and reverted to original headings from the nested list.